packages/ipfs-core/src/components/pin/remote/rm.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const withTimeoutOption = require('ipfs-core-utils/src/with-timeout-option')
+
+module.exports = ({ serviceRegistry }) => {
+  /**
+   * Remove a single pin from a remote pinning service.
+   * Fails if multiple pins match the specified query. Use rmAll to remove all pins that match.
+   *
+   * @param {Query & AbortOptions} options
+   * @returns {Promise<void>}
+   */
+  async function rm (options) {
+    const { service } = options
+    const svc = serviceRegistry.serviceNamed(service)
```

Comment:
Unless I'm missing something this will return `null` unload config was loaded which may not be the case here.